modules/tinymce/src/plugins/lists/main/ts/core/Selection.ts

```diff
@@ -67,6 +68,11 @@ const getClosestListHost = (editor: Editor, elm: Node): HTMLElement => {
   return parentBlock.getOr(editor.getBody());
 };
 
+const isListInsideLiWithSiblings = (list: SugarElement<Node>): boolean =>
+  Traverse.prevSibling(list).filter((sibling) => !NodeType.isListItemNode(sibling.dom)).isSome() &&
```

Comment:
You can reduce this by using `exists` instead of `filter` and `isSome`.

Reply:
done